gallery/src/components/NxRadio/NxRadioPage.tsx

```diff
@@ -119,11 +121,19 @@ const NxRadioPage = () =>
                         codeExamples={exampleCode}
                         liveExample={NxRadioExample}>
       This example shows a series of radios in a typical vertical layout with
-      different label content. Note that one of the radios is disabled.
+      different label content. 
```

Comment:
```suggestion
      different label content.
```

Reply:
> 

Adjusted VS Code settings and implemented here as well.
92cc934dd7e32144360bfe9850b9403c9b1da9f9